spec/requests/event_streams_spec.rb

```diff
@@ -162,4 +162,48 @@
       expect(response).to have_http_status(:forbidden)
     end
   end
+
+  describe 'OPTIONS /api/event_streams' do
+    it 'returns expected and additional attributes' do
+      options(api_event_streams_url)
+
+      expected_data = {
+        "timeline_events" => {
+          "EmsEvent" => {
+            "description"  => "Management Events",
+            "group_names"  => {
+              "addition"      => "Creation/Addition",
+              "configuration" => "Configuration/Reconfiguration",
+              "console"       => "Console Activity",
+              "deletion"      => "Deletion/Removal",
+              "general"       => "General Activity",
+              "import_export" => "Import/Export",
+              "migration"     => "Migration/Vmotion",
+              "network"       => "Network",
+              "power"         => "Power Activity",
+              "snapshot"      => "Snapshot Activity",
+              "status"        => "Status",
+              "storage"       => "Storage",
+              "update"        => "Update",
+              "firmware"      => "Firmware",
+              "devices"       => "Devices",
+              "login"         => "Login",
+              "security"      => "Security"
+            },
+            "group_levels" => {
+              "critical" => "Critical",
+              "detail"   => "Detail",
+              "warning"  => "Warning"
+            }
+          },
+          "MiqEvent" => {
+            "description"  => "Policy Events",
+            "group_names"  => {},
```

Comment:
Is this right?

Reply:
Only if I'm not seeding, let me fix that.